### PR TITLE
Release/2.1.1

### PR DESCRIPTION
**Pull request by NSeydoux**
This PR bumps the version to 2.1.1

# Release steps:

1. Inspecting the `CHANGELOG.md` to determine if the release was major, minor or patch. Make sure you align with the rest of the team first.
2. Use `npm version --no-git-tag-version <major|minor|patch>` to update `package.json`.
3. Update the `CHANGELOG.md` has been updated to show version and release date.
4. Add any `@since X.Y.Z` annotations if there have been any added or deprecated APIs.
5. Commit the changes as `release: prepare vX.Y.Z` on a branch named `release/X.Y.Z`
6. Create a PR, once CI passes, merge as a squash merge.
7. Create and publish the release via GitHub, this will create the tag and trigger the npm publish.

### CODE REVIEW

**Comment by ThisIsMissEm** [comment]
Just to confirm, before this release, `solid-client-js` was also released and updated here, right?

**Comment by NSeydoux** [comment]
Absolutely: https://github.com/inrupt/solid-client-access-grants-js/commit/701292dfdbb54dd537002a631f93dbe7f464c43b#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

**Comment by ThisIsMissEm** [comment]
> Absolutely: [701292d#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519](https://github.com/inrupt/solid-client-access-grants-js/commit/701292dfdbb54dd537002a631f93dbe7f464c43b#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519)

Okay, before we do this release, I'd like to get the two fixes out in 1.25.1 that we need to land, that way we've only gotta release this once.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
+## [2.1.1](https://github.com/inrupt/solid-client-access-grants-js/releases/tag/v2.1.1) - 2023-02-01
```
Comment (solid-akb) [inline CHANGELOG.md:7]: This changelog looks a bit different compared to the others/what I was expecting. Just want to confirm it's accurate.

**Review by solid-akb — APPROVED** [review]

**Pull request merged**